assets/js/seamless-mobilepay.js

```diff
@@ -0,0 +1,105 @@
+/* global wc_checkout_params */
+jQuery( function( $ ) {
```

Comment:
It seems like the content in most of the JavaScript files added here are 99% identical. Is it possible to generalize it a bit more so we don't end up having so much duplicated code to maintain?

Reply:
@asbjornu I've generalized that alrady :-) I moved common js code to seamless.js

Reply:
I don't understand. The contents of `assets/js/seamless-cc.js`, `assets/js/seamless-trustly.js `, etc., is almost identical.